Face-Mask 4-4 fix (#139)
neuralet/neuralet: applications/facemask/README.md

* Coral Dev Board
* AMD64 node with attached Coral USB Accelerator
* X86 node
* Jetson 

#### Software

> 

Reviewer: Can you please mention Nvidia Docker Toolkit as a requirement for training and inference on X86 nodes with GPU?

Reviewer: Also JetPack 4.4 for Jetson

Author: Sure

1- Inference on app mode (using WebGUI to show the result)

> 2- Inference on video

Reviewer: ```suggestion
2- Inference on video and save output on a video file.
```

> 3- Inference on image

Reviewer: ```suggestion
3- Inference on image and save output on a image file.
```

which are compatible with x86 devices, Coral USB Accelerator, Coral Dev Borad and Jetson.

> * NOTE: There is a config file at `configs/` directory for customizing the parameters of the model and the application. Please set the parameters if you plan to have a customized setting  

Reviewer: ```suggestion
* NOTE: There is a bunch of config files at `configs/` directory for customizing the parameters of the model and the application. Please set the parameters if you plan to have a customized setting  
```

docker run  -it --privileged -p HOST_PORT:8000 -v "$PWD/../../":/repo/ -it neuralet/face-mask:latest-coral-dev-board

# 3) Run main application python script inside the docker
python3 inference_main_app.py --config configs/config_edgetpu.json 
```
### Run on Jetson
You need to have JetPack 4.4 installed on your Jetson.
```
# 1) Build Docker image
docker build -f jetson-4-4.Dockerfile -t "neuralet/face-mask:latest-jetson-4-4" .

# 2) Run Docker container:
docker run  -it --privileged --runtime nvidia -p HOST_PORT:8000 -v "$PWD/../../":/repo/ -it neuralet/face-mask:latest-jetson-4-4

# 3) Run Image Inference inside the docker

> python3 inference_images.py --config configs/config_jetson.json --input_image_dir data/images --output_image_dir output_images

Reviewer: ```suggestion
python3 inference_main_app.py --config configs/config_jetson.json
```
